### ISSUE TITLE
query output is printed in the message window rather than the query one

### ISSUE THREAD

**Issue by Zimmi48** [Zimmi48/bugzilla-test#5684]
Note: the issue was created automatically with bugzilla2github tool

# Bugzilla Bug ID: 5684
Date: 2017-08-09 18:51:20 +0200
From: @_gares
To:   
CC:   coq-bugs-redist@lists.gforge.inria.fr, @_pstecker

Last updated: 2017-08-11 18:56:47 +0200

## Bugzilla Comment ID: 17255
Date: 2017-08-09 18:51:20 +0200
From: @_gares

Type F1, then "About nat" and you will get the answer in the wrong window.

## Bugzilla Comment ID: 17256
Date: 2017-08-09 18:52:25 +0200
From: @_gares

Assigning to Emilio since I suspect it has something to do with the feedback refactoring.

## Bugzilla Comment ID: 17264
Date: 2017-08-11 17:27:48 +0200
From: @_pstecker

Here's the communication I see in PG/xml:
--
*emacs:
<call val="Query"><pair><route_id val="1"/><pair><string>About nat .</string>
<state_id val="3"/></pair>
</pair>
</call>

*coqtop:
<feedback object="state" route="0"><state_id val="3"/><feedback_content val="processed"/></feedback>
*coqtop:
<feedback object="state" route="1"><state_id val="3"/><feedback_content val="message"><message><message_level val="notice"/><option val="none"/><richpp><_><pp>nat&nbsp;:&nbsp;<constr.type>Set</constr.type>

Expands&nbsp;to:&nbsp;Inductive&nbsp;Coq.Init.Datatypes.nat</pp></_></richpp></message></feedback_content></feedback>
*coqtop:
<value val="good"><unit/></value>
--

In PG/xml, the results show up in the *response* buffer.

So maybe CoqIDE is routing this incorrectly, or the feedback structure is wrong.

## Bugzilla Comment ID: 17265
Date: 2017-08-11 18:56:47 +0200
From: @_ejgallego

Yup, this is pending on my queue (and duplicate of some other bug), the fix should be ready before 8.7 final.
